src/test/java/org/minions/devfund/royrodriguez/WhackAMoleTest.java

```diff
@@ -0,0 +1,230 @@
+package org.minions.devfund.royrodriguez;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
```

Comment:
the correct imports are
import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;